db/schema.rb

```diff
@@ -345,15 +345,6 @@
     t.index ["issuer", "year_month", "user_id"], name: "index_monthly_auth_counts_on_issuer_and_year_month_and_user_id", unique: true
   end
 
-  create_table "monthly_sp_auth_counts", force: :cascade do |t|
-    t.string "issuer", null: false
-    t.integer "ial", limit: 2, null: false
-    t.string "year_month", null: false
-    t.integer "user_id", null: false
-    t.integer "auth_count", default: 1, null: false
-    t.index ["issuer", "ial", "year_month", "user_id"], name: "index_monthly_sp_auth_counts_on_issuer_ial_month_user_id", unique: true
-  end
-
```

Comment:
Is it expected that we're removing this table? I'm not following how this is related to the other work.

In any case, if so, would we also want to remove `spec/factories/monthly_sp_auth_counts.rb`, or any/all of the `MonthlySpAuthCount` module code?